docs/book/src/reference/samples.md

```diff
@@ -0,0 +1,128 @@
+```
+apiVersion: cluster.x-k8s.io/v1alpha4
```

Comment:
This could cause issues as the file will be out of date very quickly as CAPI/Kubernetes continues to evolve. I think the solution here is to either add some sort of automation of the creation of this file or maybe use some of the minimal examples we have under [tests/infrastructure/docker/examples](https://github.com/kubernetes-sigs/cluster-api/blob/master/test/infrastructure/docker/examples/simple-cluster.yaml) At least then we keep the update problem in the same location.